projects/demo/src/modules/components/link/index.html

```diff
@@ -10,6 +10,7 @@
             *ngFor="let example of examples; let index = index"
             [component]="index + 1 | tuiComponent"
             [content]="index + 1 | tuiExample: 'html,ts'"
+            [fullsize]="index === 4"
```

Comment:
Let's not do that, instead add `width: max-content;` to the `div` inside example